### PR TITLE
[ES] Add `Skinning/FAQ`

### PR DESCRIPTION
**Pull request by ImMock**
Translation for the spanish language, ready for review

### CODE REVIEW

File: wiki/Skinning/FAQ/es.md [hunk 1]
```diff
@@ -0,0 +1,294 @@
+# Skinning FAQ
+
+Este artículo trata de las preguntas frecuentes sobre skinning.
+
+## Comenzando
+
+### ¿Qué es skinning?
+
+Skinning es un simple mecanismo de cambio de imágenes que osu! cargara después de seleccionar una skin en las opciones. Montar una skin es fácil, pero crear los archivos para esta, es un poco más difícil de lograr.
```
Comment (Repflez) [inline wiki/Skinning/FAQ/es.md:9]:
```suggestion
Skinning es un simple mecanismo de cambio de imágenes que osu! cargará después de seleccionar una skin en las opciones. Montar una skin es fácil, pero crear los archivos para esta, es un poco más difícil de lograr.
```

File: wiki/Skinning/FAQ/es.md [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+
+### ¿Puede alguien hacer una skin de esa serie/juego?
+
+Si has recorrido todos los foros de skinning y estás seguro de que no puedes encontrar *esa* skin,  entonces felicidades, osu! aun no cuenta con ella. ¡Con esto en mente, puedes tomar esta iniciativa para crear la skin de tus sueños y hacerla de manera y hacerla algo que todos los demás puedan querer!
+
+**Sin embargo, por favor, nunca solicites una skin en ningún lugar de los foros.** El incumplimiento de esta regla hará que tu hilo sea trasladado a los terrenos baldíos.
```
Comment (Repflez) [inline wiki/Skinning/FAQ/es.md:15]:
```suggestion
**Sin embargo, por favor, nunca solicites una skin en ningún lugar de los foros.** El incumplimiento de esta regla hará que tu hilo sea trasladado al olvido.
```

Abandoned Wasteland is a name here, no need to translate it directly.

File: wiki/Skinning/FAQ/es.md [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+
+### ¿Dónde puedo conseguir esta skin la cual vi a una persona jugar con ella?
+
+En primer lugar, no preguntes esto en los foros; de lo contrario, lo más probable es que tu hilo sea trasladado a los terrenos baldíos. En segundo lugar, es mejor que le pidas a esa persona la skin que está utilizando. Puede que te la den, pero no siempre será el caso.
```
Comment (Repflez) [inline wiki/Skinning/FAQ/es.md:19]:
```suggestion
En primer lugar, no preguntes esto en los foros; de lo contrario, lo más probable es que tu hilo sea trasladado al olvido. En segundo lugar, es mejor que le pidas a esa persona la skin que está utilizando. Puede que te la den, pero no siempre será el caso.
```

Same as above

File: wiki/Skinning/FAQ/es.md [hunk 4]
```diff
@@ -0,0 +1,294 @@
...
+2. La imagen estática en el directorio raíz.
+3. La imagen predeterminada.
+
+*Aviso: En osu!mania, osu! ignora la ruta especificada en el archivo `skin.ini` y solo usara la imagen del directorio raíz (la carpeta donde los elementos de la skin son cargados).*
+
+## Animaciones
+
+### ¿Cómo se anima un elemento?
+
+Primero, revisa los archivos listados para verificar que el archivo que desees animar pueda ser animado. Si es posible, entonces podrás comenzar la animación. Para poder hacer esto, necesitaras que cada fotograma este en imágenes separadas. Cuando cuentes con esto, nombre cada archivo, seguido de un guion (`-`), entonces una cuenta de índice (comenzando de 0). Repite esto por el resto de fotogramas.
```
Comment (Repflez) [inline wiki/Skinning/FAQ/es.md:219]:
```suggestion
Primero, revisa los archivos listados para verificar que el archivo que desees animar pueda ser animado. Si es posible, entonces podrás comenzar la animación. Para poder hacer esto, necesitarás que cada fotograma este en imágenes separadas. Cuando cuentes con esto, nombre cada archivo, seguido de un guion (`-`), entonces una cuenta de índice (comenzando de 0). Repite esto por el resto de fotogramas.
```

File: wiki/Skinning/FAQ/es.md [hunk 5]
```diff
@@ -0,0 +1,294 @@
...
+
+### Eliminé un elemento, pero sigue ahí. ¿Cómo puedo removerlo?
+
+Si deseas remover un elemento, deberás utilizar una imagen transparente en el elemento que deseas remover. Entonces, [recarga tu skin](#¿Cómo-puedo-recargar-mi-skin-en-osu!?).
+
+#### ¿Cómo puedo eliminar la iluminación después de pulsar un objeto?
+
+Hay dos maneras de hacer esto:
+
+- Desactivar e; `Hit Lighting` en las [opciones](/wiki/Client/Options).
```
Comment (Repflez) [inline wiki/Skinning/FAQ/es.md:251]:
```suggestion
- Desactivar el `Hit Lighting` en las [opciones](/wiki/Client/Options).
```

**Review by Repflez — CHANGES_REQUESTED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**